OracleLinuxDevelopers/oraclelinux9/httpd/2.4/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eu pipefail
+
+SERVER_CERT="$CERTIFICATE_DIR/localhost.crt"
+SERVER_KEY="$KEY_DIR/localhost.key"
```

Comment:
We should also check for the existance of the standard ACME files, e.g. `privkey.pem`, `cert.pem`, `fullchain.pem` and `ca.pem`.

Or just rename these two to `privkey.pem` and `fullchain.pem` so they can be overloaded via bind mount to a host-wide ACME certificate.

Reply:
This is just the default for the SSL config - I could do a sed to fix it though. Expecting acme mounts seems logical.

Reply:
Also, some sort of output to `stderr` if we do fallback to self-signed certificates would probably be pretty useful too.